openslides/users/static/js/users/site.js

```diff
@@ -312,26 +322,6 @@ angular.module('OpenSlidesApp.users.site', ['OpenSlidesApp.users'])
                     }
                 },
                 {
-                    key: 'default_password',
```

Comment:
in the users list is a new link "change password" in the hover menu. It is a suggestion to solve it like this, but there are some reasons:

- the default password is a little bit confusing, so is the reset button. 
- changing a password is not trivial - there should be a seperate place to do this, because the administrator should be aware of, what he is doing.
- the behavior of the reset button is not clear for an unexperienced user. Why do you have to click reset and could not simply save?

@emanuelschuetze has some more arguments ;)